airbyte-integrations/connectors/source-gcs/integration_tests/spec.json

```diff
@@ -437,12 +437,83 @@
           "required": ["name", "format"]
         }
       },
-      "service_account": {
-        "title": "Service Account Information",
-        "description": "Enter your Google Cloud <a href=\"https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys\">service account key</a> in JSON format",
-        "airbyte_secret": true,
+      "credentials": {
+        "title": "Authentication",
+        "description": "Credentials for connecting to the Google Cloud Storage API",
+        "type": "object",
         "order": 0,
-        "type": "string"
+        "discriminator": {
```

Comment:
I expect the discriminator fields to trip up the Terraform provider. Maybe we need a method similar to this one where we remove them from the schemas https://github.com/airbytehq/airbyte/blob/fc12432305da7725379f967a64d0b0f011e02d78/airbyte-cdk/python/airbyte_cdk/sources/file_based/config/abstract_file_based_spec.py#L58-L60

Reply:
removed `def remove_discriminator` from Config class, so both discriminators are being removed from spec